packages/web-app-files/src/components/SideBar/Exif/ExifPanel.vue

```diff
@@ -158,12 +158,19 @@ export default defineComponent({
 </script>
 
 <style lang="scss" scoped>
+#files-sidebar-panel-exif {
+  background-color: var(--oc-role-surface-container);
+  border-radius: 5px;
```

Comment:
You could use `oc-rounded` as a class on the panel instead of hardcoding the pixels here.